content/docs/app-developer-guide/build-a-windows-app.md

```diff
@@ -0,0 +1,87 @@
+> - If you encounter any problems while experimenting, we'd love for you to let us know by filing an issue on the [pack][pack-issues] or [lifecycle][lifecycle-issues] repo.
+
+### Recommended reading
+
+Before trying out builds for Windows images, we recommend following the [Linux image tutorial][app-journey] under [Getting Started][getting-started]. Don't worry, you can still run it on a Windows OS if you need to -- just make sure to [enable Linux containers][container-mode] for Docker first.
+
+When you're done, head back here.
+
+### Understanding Docker hosts
+
+The `pack` CLI supports using the [`DOCKER_HOST`][docker-env-vars] environment variable. Essentially, this points your local Docker client (e.g. `docker` or `pack` CLI) to a remote Docker daemon. For instance, if the IP address of your host is `192.168.2.100`, you can set `DOCKER_HOST` to `tcp://192.168.2.100`. Any subsequent `pack` or `docker` commands would communicate with the daemon on that machine instead of a local daemon.
+
+By setting `DOCKER_HOST`, you can use `pack` on any OS to build Windows container images, as long as your remote Docker host is configured to support them (i.e. the host runs a Windows OS and has [Windows container mode enabled][container-mode]).
+
+> **NOTE**: When setting `DOCKER_HOST`, keep in mind that:
+>
+> - any volumes mounted via `pack build --volume <volume> ...` or `docker run --volume <volume> ...` must exist on the _docker host_, not the client machine.
+> - any ports published via `docker run --publish <host-port>:<container-port> ...` will be published on the _docker host_, not the client machine.
+
+### Enable Windows container mode
+
+In order to produce Windows container images, ensure [Windows container mode][container-mode] is enabled in your Docker settings (available only in Docker for Windows). See [Understanding Docker hosts][docker-hosts]) above if you're using a remote Docker host.
+
+Then, building a Windows app using Cloud Native Buildpacks is nearly identical to [building for Linux][build-linux]:
+
+### 1. Select a builder
+
+To [build][build] an app you must first decide which [builder][builder] you're going to use. A builder
+includes the [buildpacks][buildpack] that will be used as well as the environment for building your
+app.
+
+For this guide we're going to use a sample builder, `cnbs/sample-builder:dotnet-framework-1809`.
+
+### 2. Build your app
+
+Now we can build our app. For this example we'll use our [samples][samples] repo for simplicity.
+
+```bash
+# clone the repo
+git clone https://github.com/buildpacks/samples
+
+# build the app
+pack build sample-app --path samples/apps/aspnet --builder cnbs/sample-builder:dotnet-framework-1809 --trust-builder
+```
+
+> **TIP:** If you don't want to keep specifying a builder every time you build, you can set it as your default
```

Comment:
In case you're wondering why I changed `Tip` to `TIP` elsewhere in this PR, it felt weird to leave it lowercase here, and it felt weird to not make it consistent elsewhere.

That said, we could also change `NOTE` to `Note` everywhere instead. It may be more consistent that way, as the installation docs do use `Optional:` (vs `OPTIONAL`).